packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -57,73 +67,13 @@ function ListView() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [canRead]);
 
-  const getWebhookIndex = id => webhooks.findIndex(webhook => webhook.id === id);
-
-  // New button
-  const addBtnLabel = formatMessage({
-    id: 'Settings.webhooks.list.button.add',
-  });
-
-  const newButtonProps = {
-    label: addBtnLabel,
-    onClick: () => handleGoTo('create'),
-    color: 'primary',
-    type: 'button',
-    icon: <Plus fill="#007eff" width="11px" height="11px" />,
-    Component: props => {
-      if (canCreate) {
-        return <Button {...props} />;
-      }
-
-      return null;
-    },
-  };
-
-  // Header props
-  const actions = [
-    {
-      ...newButtonProps,
-      icon: true,
-      style: {
-        paddingLeft: 15,
-        paddingRight: 15,
-      },
-    },
-  ];
-
   const headerProps = {
     title: {
       label: formatMessage({ id: 'Settings.webhooks.title' }),
     },
     content: formatMessage({ id: 'Settings.webhooks.list.description' }),
-    actions,
-  };
-
-  // List props
-  const rowsCount = webhooks.length;
-  const titleLabel = `${
-    rowsCount > 1
-      ? formatMessage({ id: 'Settings.webhooks.title' })
-      : formatMessage({ id: 'Settings.webhooks.singular' })
-  }`;
-  const title = `${rowsCount} ${titleLabel}`;
-
-  /* eslint-disable indent */
-  const deleteButtonProps = canDelete
-    ? {
-        color: 'delete',
-        disabled: !(webhooksToDelete.length > 0),
-        label: formatMessage({ id: 'app.utils.delete' }),
-        onClick: () => setShowModal(true),
-        type: 'button',
-      }
-    : null;
-  /* eslint-enable indent */
-
-  const listProps = {
-    title,
-    button: deleteButtonProps,
-    items: webhooks,
+    addBtnLabel: formatMessage({ id: 'Settings.webhooks.list.button.add' }),
```

Comment:
Could you please pass all the props of this variable directly to the component that is concerned?

Instead of 

```jsx
const headeProps = {
  addBtnLabel: : formatMessage({ id: 'Settings.webhooks.list.button.add' })
}

// and later on

<Button>
{headerProps.addBtnLabel}
</Button>
```

Could you try doing directly:

```jsx
<Button>{formatMessage({ id: 'Settings.webhooks.list.button.add' })}</Button>
```

please 🙏🏻 😊